parameters.yaml

```
@@ -0,0 +1,32 @@
+# parameters.yaml
+
+# Parameters file format version 0.1
+# This is a draft YAML file for data in a single experiment run
+# Subject to changes as needed
+
+paths:
+    base: <To be set to the local computer's data directory>
+    protocol: HB_139_DAMGO/
+    unit: 825_12h24m37s/20200825_12h24m37s_units_ts.mat
+    tx: 825_12h24m37s/20200825_12h24m37s_treatments.csv
```

"treatments" instead of "tx"

---

Yes "treatments" is better. more clear.
```suggestion
    treatments: 825_12h24m37s/20200825_12h24m37s_treatments.csv
```